src/features/publishDebugInfo/Component.tsx

```diff
@@ -159,18 +163,15 @@ class PublishDebugLogModal extends Component {
             <p className={classes.info}>
               {intl.formatMessage(messages.published)}
             </p>
-            <Input showLabel={false} value={`${DEBUG_API}/${log}`} readonly />
+            <Input showLabel={false} value={`${DEBUG_API}/${log}`} />
```

Comment:
Why remove the `readonly`? That would make this a text box that the user can type into. If this ts-conversion complains, then can we convert this into a label?

Reply:
readOnly is allowed by TS as we extending a InputHTMLAttributes props
but it not used inside Input tag 
I shall leave a comment for now, will check it later